utils/constants.tsx

```diff
@@ -677,8 +679,8 @@ export const DefaultRolePermissions = {
         Permissions.MANAGE_PRIVATE_CHANNEL_MEMBERS,
         Permissions.DELETE_POST,
         Permissions.EDIT_POST,
-        Permissions.MANAGE_CREATE_EMOJIS,
-        Permissions.MANAGE_DELETE_EMOJIS,
+        Permissions.CREATE_EMOJIS,
+        Permissions.DELETE_EMOJIS,
```

Comment:
This change is based on the fact that at the time these two lines were added (and now as well), there were no such permissions in `mattermost-redux/src/constants/permissions.js` and no such permissions were referenced in `mattermost-web` nor `mattermost-redux`.

Commits that introduced this change:
- mattermost-webapp: [Using new splited emojis and webhooks permissions (#2339)](https://github.com/mattermost/mattermost-webapp/commit/cc2d3b6f34aafdb376a78675ddd6229f652c8629)
- from mattermost-redux [Splitting emojis and webhooks permissions (#4532)](https://github.com/mattermost/mattermost-redux/commit/e7a6dea5629281326e0e17a8faadccac8abb3120)

Reply:
Updating `mattermost-redux` to the latest commit and updating your branch to `master` should resolve the need for this.

Reply:
Hmm, I've fetched master for both `mattermost-webapp` and `mattermost-redux` and I've run `npm update` afterwards, but this line still gives me an error `TS2551: Property 'MANAGE_CREATE_EMOJIS' does not exist on type (...)` and same for `MANAGE_DELETE_EMOJIS`. I also don't see them in [mattermost-redux/src/constants/permissions.ts](https://github.com/mattermost/mattermost-redux/blob/master/src/constants/permissions.ts).

Reply:
@jespino Should `utils/constants.tsx` be updated on `master` here?

Reply:
Yes, I think so, I think the current state is a bug, and we should change that.